### PR TITLE
Update tabbed-conceptual.md

### PR DESCRIPTION
**Pull request by meganbradley**

### CODE REVIEW

**Comment by meganbradley** [comment]
@mairaw @scottaddie sure, you can add the additional requests to the same DCR. I'll wait until it's all done to merge.

**Comment by scottaddie** [comment]
@meganbradley I've added the 3 tabs to the DCR.

**Comment by meganbradley** [comment]
Also added these to this doc:
[MSTest](#tab/mstest)
[NUnit](#tab/nunit)
[xUnit](#tab/xunit)

File: Contribute/validation-reference/tabbed-conceptual.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ The following tab name/tab ID pairs are whitelisted. Dependent tab IDs are not p
 |`[.NET Core 2.2]`     |`(#tab/netcore22)`|
 |`[.NET Core 3.x]`     |`(#tab/netcore3x)`|
 |`[.NET Core 3.0]`     |`(#tab/netcore30)`|
+|`[.NET Core 3.0 Preview 1]` |`(#tab/netcore30)`|
```
Comment (scottaddie) [inline Contribute/validation-reference/tabbed-conceptual.md]: @meganbradley Are tab IDs supposed to be unique? I noticed this has the same ID as the tab above it.

Comment (meganbradley) [reply Contribute/validation-reference/tabbed-conceptual.md]: We have an open question on that. @TinaLiuGirl can you confirm whether tab IDs need to be unique?

Comment (JasonWHowell) [reply Contribute/validation-reference/tabbed-conceptual.md]: @meganbradley @TinaLiuGirl @scottaddie  Any update on this old open discussion?

**Comment by BillWagner** [comment]
@meganbradley should this just be closed now? Is it still relevant?

**Comment by scottaddie** [comment]
It's relevant if zone pivots aren't replacing tabs. Any updates on that initiative?

**Pull request closed without merging**